alert/search.go

```diff
@@ -69,7 +73,11 @@ var searchTemplate = template.Must(template.New("search").Parse(`
 		AND a.status = any(:status::enum_alert_status[])
 	{{ end }}
 	{{ if .Services }}
-		AND a.service_id = any(:services)
+		AND (a.service_id = any(:services)
+			{{ if .NotifiedUserID }}
+				OR a.id = any(select alert_id from alert_logs where event = 'notification_sent' and sub_user_id = :currentUserID)
```

Comment:
One additional thought for this, we can add `no_notification_sent` for users who were on-call but misconfigured.
```suggestion
				OR a.id = any(select alert_id from alert_logs where event in ('notification_sent', 'no_notification_sent') and sub_user_id = :currentUserID)
```

Reply:
True, I'll go ahead and update the alerts list text to cover this as well 👍